src/qt/transactiondesc.cpp

```diff
@@ -167,9 +146,6 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx, TransactionReco
 
         if (fAllFromMe)
         {
-            if(fAllFromMe & ISMINE_WATCH_ONLY)
-                strHTML += "<b>" + tr("From") + ":</b> " + tr("watch-only") + "<br>";
```

Comment:
Also isn't this part of code inefficient?
```c
if (fAllFromMe)
         {
           if(fAllFromMe & ISMINE_WATCH_ONLY)
```
I am going to replace this with this:
```c
if (fAllFromMe)
         {
           if(ISMINE_WATCH_ONLY)
```